tests/avifyuv.c

```diff
@@ -468,21 +468,22 @@ int main(int argc, char * argv[])
                     }
                 }
 
+                rgb.width = a + 1;
```

Comment:
In line 443 above, rgb.width is set to 'size' so that the avifRGBImageAllocatePixels() call at line 451 allocates an rgb.pixels buffer large enough for all the tests. Here, we set rgb.width to the actual width of the current test.